java/semantickernel-api/pom.xml

```diff
@@ -39,4 +39,11 @@
         </dependency>
     </dependencies>
 
+    <build>
+        <resources>
+            <resource>
+                <directory>src/main/java/com/microsoft/semantickernel/tokenizers/settings/resources</directory>
```

Comment:
this should remain at default(src/main/resources), your later files should be at:

```
src/main/resources/com/microsoft/semantickernel/tokenizers/settings/encoder.json
src/main/resources/com/microsoft/semantickernel/tokenizers/settings/vocab.pbe
```